modules/updating-eus-to-eus-upgrade.adoc

```diff
@@ -11,7 +11,12 @@ Following this procedure reduces the total update duration and the number of tim
 
 .Prerequisites
 
-* Review the release notes for {product-title} <4.y+1> and <4.y+2>
+* Review the release notes for {product-title} <4.y+1> and <4.y+2>.
 * Review the release notes and product lifecycles for any layered products and Operator Lifecycle Manager (OLM) Operators. Some may require updates either before or during an EUS-to-EUS update.
 * Ensure that you are familiar with version-specific prerequisites, such as the removal of deprecated APIs, that are required prior to updating from {product-title} <4.y+1> to <4.y+2>.
-
+* If your cluster uses vSphere, update vSphere to version 7.0u3L+ or 8.0u2+.
```

Comment:
I'd expect something closer to [the admin-ack/`Upgradeable=False` message][1]:

> vSphere CSI migration will be enabled in Openshift-4.14. Your cluster appears to be using in-tree vSphere volumes and is on a vSphere version that has CSI migration related bugs. See - https://access.redhat.com/node/7011683 for more information, before upgrading to 4.14.

Important details from that message:

* The difficult spot is the transition from 4.13 to 4.14, not 4.12 to 4.13.  In these 4.12-to-4.13-to-4.14 docs, I'm fine skipping this if you think it's not worth the space.
* The reason is a vSphere CSI migration, and only clusters that use in-tree vSphere volumes are exposed.  This seems useful, but we can skip it if we expect most/all vSphere clusters to use volumes.
* https://access.redhat.com/node/7011683 has more details.  This seems like it's worth linking, even if you leave your current text unchanged.

[1]: https://github.com/openshift/vmware-vsphere-csi-driver-operator/pull/171/files#diff-bd7b198fe282f7e27754079508639882be175cc903f563fb69f2c89fc6875a2aR520

Reply:
+1 i think it's best to add a link to the KCS

Reply:
I'm okay with adding a link to the KCS. Is it fine if I put in in the admonition below as context for the "known issues [that] may occur"? Or is it critical to add it right in the bullet point?

Otherwise I think if a lot of this other context is covered in the KCS, then I'd prefer not to bloat up this prereq item with redundant context. I was aiming to write this prereq as "just do this prereq if you use vSphere, here's a little context, don't say we didn't warn you" in a sentence or two, rather than adding a paragraph of info in the prereqs.